### PR TITLE
New single scenario

### PR DESCRIPTION
**Pull request by LukeLoze1**

### CODE REVIEW

**Review by hbrace — APPROVED** [review]

File: utilities/collection_exercise_utilities.py [hunk 1]
```diff
@@ -8,18 +8,28 @@
...
-        "userDescription": user_description
+        "surveyRef": survey_ref,
+        "name": survey_ref[:20],
+        "exerciseRef": '1',
+        "userDescription": '1',
     }
-    response = requests.post(create_collection_exercise_url, auth=Config.BASIC_AUTH, json=collex)
-    response.raise_for_status()
-    response_json = response.json()
-    logger.debug("Successfully created collection exercise", exercise_ref=exercise_ref)
```
Comment (rjweeks) [inline utilities/collection_exercise_utilities.py:21]: We still want the logging.

File: utilities/collection_exercise_utilities.py [hunk 2]
```diff
@@ -8,18 +8,28 @@
...
-def create_collection_exercise(survey_id, exercise_ref, user_description):
-    create_collection_exercise_url = f'{Config.COLLECTION_EXERCISE}/collectionexercises'
+def create_collection_exercise(survey_ref):
     collex = {
-        "surveyId": survey_id,
-        "exerciseRef": exercise_ref,
-        "userDescription": user_description
+        "surveyRef": survey_ref,
+        "name": survey_ref[:20],
+        "exerciseRef": '1',
```
Comment (rjweeks) [inline utilities/collection_exercise_utilities.py:15]: Why have you hard coded exerciseRef and userDescription? These should be parameters.

File: utilities/collection_exercise_utilities.py [hunk 3]
```diff
@@ -8,18 +8,28 @@
...
-    response = requests.post(create_collection_exercise_url, auth=Config.BASIC_AUTH, json=collex)
-    response.raise_for_status()
-    response_json = response.json()
-    logger.debug("Successfully created collection exercise", exercise_ref=exercise_ref)
-    return response_json
+
+    response = requests.post(f"{Config.COLLECTION_EXERCISE_SERVICE}/collectionexercises",
+                             auth=Config.BASIC_AUTH, json=collex)
+
+    assert response.status_code == 201
```
Comment (rjweeks) [inline utilities/collection_exercise_utilities.py]: Use response.raise_for_status not asserts in this utility code.

File: utilities/survey_utilities.py [hunk 4]
```diff
@@ -30,3 +30,14 @@ def create_survey(survey_ref, short_name, long_name, legal_basis='GovERD', surve
     # create_classifiers(response_json['id'])
     logger.debug("Successfully created survey", short_name=short_name)
     return response_json
+
+
+def create_survey_classifier(survey_id):
+    post_survey_classifiers = f'{Config.SURVEY_SERVICE}/surveys/{survey_id}/classifiers'
+    classifiers = {"name": "COLLECTION_INSTRUMENT", "classifierTypes": ["COLLECTION_EXERCISE"]}
+    response = requests.post(post_survey_classifiers, auth=Config.BASIC_AUTH, json=classifiers)
+    assert response.status_code == 201
```
Comment (rjweeks) [inline utilities/survey_utilities.py]: I don't think we should use asserts in Utility code. Use response.raise_for_status

**Review by rjweeks — CHANGES_REQUESTED** [review]

File: utilities/collection_exercise_utilities.py (hunk 2 above)
Comment (LukeLoze1) [reply utilities/collection_exercise_utilities.py:15]: The idea is to get something working for the 1st simple test.  If we want to parameterize it further for later tests then that's good

File: utilities/collection_exercise_utilities.py (hunk 1 above)
Comment (LukeLoze1) [reply utilities/collection_exercise_utilities.py:21]: done

File: utilities/collection_exercise_utilities.py [hunk 5]
```diff
@@ -8,18 +8,30 @@
...
```
Comment (hbrace) [inline utilities/collection_exercise_utilities.py]: Should be response.raise_for_status()? Asserts in the test steps?

File: utilities/survey_utilities.py (hunk 4 above)
Comment (hbrace) [inline utilities/survey_utilities.py]: response.raise_for_status()?

File: utilities/collection_exercise_utilities.py (hunk 5 above)
Comment (LukeLoze1) [reply utilities/collection_exercise_utilities.py]: done

**Review by rjweeks — APPROVED** [review]

**Comment by rjweeks** [comment]
[Automated testing](https://trello.com/c/PUT3GUaJ/438-automated-testing)

**Pull request merged**